src/status_im/contacts/handlers.cljs

```diff
@@ -271,13 +271,23 @@
           (dispatch [:update-contact! {:whisper-identity from
                                        :last-online      timestamp}]))))))
 
-(register-handler :remove-contact
+(register-handler :hide-contact
   (after stop-watching-contact)
   (u/side-effect!
     (fn [_ [_ {:keys [whisper-identity] :as contact}]]
       (dispatch [:update-contact! (assoc contact :pending true)])
       (dispatch [:account-update-keys]))))
 
+(register-handler :remove-contact
+  (fn [db [_ whisper-identity cond]]
```

Comment:
IMO, better to use, for instance, `pred` instead of `cond` because we have standard function with the same name